src/test/java/com/hillel/items_exchange/controller/UserControllerTest.java

```diff
@@ -111,20 +111,20 @@ void updateUserInfo_shouldReturn403WhenUsernameIsChanged() throws Exception {
     @WithMockUser(username = "admin")
     @Transactional
     @DataSet("database_init.yml")
-    void updateUserInfo_shouldBeThrownAccessDeniedExceptionWhenNotUserId() throws Exception {
+    void updateUserInfo_shouldReturn403WhenLastOnlineTimeIsChanged() throws Exception {
         MvcResult result = getResultActions(HttpMethod.PUT, "/user/info",
-                createUserDtoForUpdatingWithNotUserId(), status().isConflict())
+                createUserDtoForUpdatingWithChangedLastOnlineTimeWithoutChildrenOrPhones(), status().isForbidden())
                 .andReturn();
-        assertTrue(result.getResponse().getContentAsString().contains("You are not allowed to perform this action"));
+        assertTrue(result.getResponse().getContentAsString().contains("You are unable to change your: LastOnlineTime"));
```

Comment:
You could export such message into messages.properties and get from it. So you won't need to update it every time it was updated there.